ci/dockerfiles/rc/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:bionic AS ubuntu
+
+FROM ubuntu AS assets
+COPY ./rc/*.tgz /tmp
+RUN tar xzf /tmp/*tgz -C /usr/local
+
+
+FROM ubuntu
+
+# 'web' keys
+ENV CONCOURSE_SESSION_SIGNING_KEY     /concourse-keys/session_signing_key
+ENV CONCOURSE_TSA_AUTHORIZED_KEYS     /concourse-keys/authorized_worker_keys
+ENV CONCOURSE_TSA_HOST_KEY            /concourse-keys/tsa_host_key
+
+# 'worker' keys
+ENV CONCOURSE_TSA_PUBLIC_KEY          /concourse-keys/tsa_host_key.pub
+ENV CONCOURSE_TSA_WORKER_PRIVATE_KEY  /concourse-keys/worker_key
+
+# enable DNS proxy to support Docker's 127.x.x.x DNS server
+ENV CONCOURSE_GARDEN_DNS_PROXY_ENABLE         true
+ENV CONCOURSE_WORKER_GARDEN_DNS_PROXY_ENABLE  true
+
+# auto-wire work dir for 'worker' and 'quickstart'
+ENV CONCOURSE_WORK_DIR                /worker-state
+ENV CONCOURSE_WORKER_WORK_DIR         /worker-state
+
+# location for base resource types
+ENV CONCOURSE_RESOURCE_TYPES          /usr/local/concourse/resource-types
```

Comment:
this can be removed - the binary auto-discovers it:

https://github.com/concourse/concourse/blob/74730b03b769ae2164bae19747d326ca354faddc/bin/cmd/concourse/worker.go#L66

Reply:
Aaah nice, thx!! Updated!